prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml

```diff
@@ -1,3 +1,32 @@
+postsubmits:
+  GoogleCloudPlatform/compute-image-tools:
+  # Integ tests that need credentials to create resources in a GCP project.
+  - name: cli-tools-postsubmit-module-tests
+    cluster: gcp-guest
+    decorate: true
+    annotations:
+      testgrid-dashboards: googleoss-gcp-guest-import-export
+      testgrid-tab-name: module-tests
+      description: "Postsubmit tests; executes cli_tools_tests/module/"
+    branches:
+      - ^master$
+    labels:
+      preset-daisy: "true"
+    spec:
+      containers:
+        - image: gcr.io/gcp-guest/gotest:latest
+          imagePullPolicy: Always
+          command:
+            - "/go/main.sh"
+          args:
+           - "cli_tools_tests/module/"
+           - "-test.timeout=30m"
```

Comment:
Added support for extra test args here: https://github.com/GoogleCloudPlatform/guest-test-infra/pull/307